scripts/generate-app-config/generatePagesConfig.js

```diff
@@ -1,4 +1,4 @@
-const path = require('path');
+const path = require('path').posix;
```

Comment:
Down on [line 219 in this file] (https://github.com/cerner/terra-dev-site/pull/81/files#diff-92bf9f0a216864327aecb81490a4575aR219) I'm getting a null reference exception, after debugging I found out that 
`
entrypoint  = "C:\terra-core/packages/[^/]*/src/terra-dev-site"
filepath = "C:/terra-core/packages/terra-abstract-modal/src/terra-dev-site/doc/AbstractModal.doc.jsx"
`
and thus weren't matching since they are using different path separators on the disk drive separator.

Reply:
glob must return the windows path and not the posix path